server-spi/src/main/java/org/keycloak/storage/user/UserQueryProvider.java

```diff
@@ -144,6 +145,10 @@ default int getUsersCount(RealmModel realm, boolean includeServiceAccount) {
     }
 
     List<UserModel> getUsers(RealmModel realm);
+    default Stream<UserModel> getUsersStream(RealmModel realm) {
```

Comment:
Are we going to add javadoc for the default methods?

Reply:
I will be adding all Javadocs in https://issues.redhat.com/browse/KEYCLOAK-15524. I will start working on that after the streamification is done. Anyway, I am about to remove these methods from interfaces, so that we avoid confusion.